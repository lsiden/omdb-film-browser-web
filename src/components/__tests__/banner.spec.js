import React from "react"
import { shallow } from "enzyme"

import Banner from "components/banner"

const createWrapper = () => shallow(<Banner />)

test("click invokes onClick()", () => {
  expect(createWrapper()).toMatchSnapshot()
})
