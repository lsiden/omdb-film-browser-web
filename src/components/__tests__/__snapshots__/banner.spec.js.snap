// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`click invokes onClick() 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Banner />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          style={
                    Object {
                              "fontSize": "18pt",
                              "marginBottom": "18pt",
                            }
          }
>
          Search Open Movie Database
           
          <a
                    href={undefined}
                    style={
                              Object {
                                        "color": "white",
                                        "fontSize": "small",
                                      }
                    }
          >
                    (OMDB)
          </a>
</h1>,
        <Connect(QueryForm) />,
      ],
      "style": Object {
        "background": "linear-gradient(to right, #000040, #3b00ff)",
        "color": "white",
        "padding": "10px",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Search Open Movie Database",
            " ",
            <a
              href={undefined}
              style={
                            Object {
                                          "color": "white",
                                          "fontSize": "small",
                                        }
              }
>
              (OMDB)
</a>,
          ],
          "style": Object {
            "fontSize": "18pt",
            "marginBottom": "18pt",
          },
        },
        "ref": null,
        "rendered": Array [
          "Search Open Movie Database",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "(OMDB)",
              "href": undefined,
              "style": Object {
                "color": "white",
                "fontSize": "small",
              },
            },
            "ref": null,
            "rendered": "(OMDB)",
            "type": "a",
          },
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            style={
                        Object {
                                    "fontSize": "18pt",
                                    "marginBottom": "18pt",
                                  }
            }
>
            Search Open Movie Database
             
            <a
                        href={undefined}
                        style={
                                    Object {
                                                "color": "white",
                                                "fontSize": "small",
                                              }
                        }
            >
                        (OMDB)
            </a>
</h1>,
          <Connect(QueryForm) />,
        ],
        "style": Object {
          "background": "linear-gradient(to right, #000040, #3b00ff)",
          "color": "white",
          "padding": "10px",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Search Open Movie Database",
              " ",
              <a
                href={undefined}
                style={
                                Object {
                                                "color": "white",
                                                "fontSize": "small",
                                              }
                }
>
                (OMDB)
</a>,
            ],
            "style": Object {
              "fontSize": "18pt",
              "marginBottom": "18pt",
            },
          },
          "ref": null,
          "rendered": Array [
            "Search Open Movie Database",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "(OMDB)",
                "href": undefined,
                "style": Object {
                  "color": "white",
                  "fontSize": "small",
                },
              },
              "ref": null,
              "rendered": "(OMDB)",
              "type": "a",
            },
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
